src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -554,6 +554,20 @@ private void commitFile() throws IOException {
     }
   }
 
+  private ProducerRecord<String, Object> createLogRecord(TopicPartition tp, String dir, String file) {
+    String topic = tp.topic() + "-log";
```

Comment:
As we discussed different connectors can export data from same topics to different locations for different purposes. So name of log topic should contain name of connector as well. It would be nice to generalise this and expose name of log topic as configurations of connector with substitute values. Something like:
```
hdfs.writer.logging.topic_name_template = '$connector_$topic_log'
```

where $connector and $topic would be substituted by our connector implementation to actual values at runtime?